### ISSUE TITLE
Deprecation Note

### ISSUE THREAD

**Issue by Lerentis** [Lerentis/terraform-provider-gitea#58]
Hello everyone who uses this terraform provider 👋 

tl;dr Deprecate this provider in favor of the official gitea terraform provider. Migration guide will follow

When i was starting to work on this terraform provider i only wanted to add one feature, but found it to be fun to explore and learn the internals of terraform as well as to work with the gitea go sdk. I never expected more people to have a need or use this provider other than me as there were not many in the terraform registry and most of them did not really support anything interesting when it comes to managing a gitea instance.
Further down the road i learned that simply nobody had the time to write this provider before and that i simply filled a gap. This was a great realisation and straightened my believe in open source and that this is the way technology should evolve.

I [proposed my changes upstream to the official gitea provider](https://gitea.com/gitea/terraform-provider-gitea/pulls/2) which has at that time not been published to the terraform registry and it got merged with the message that it will be published soon. a few month later the official gitea provider has been published to the terraform registry. 
Rather fast [i proposed to merge](https://github.com/go-gitea/terraform-provider-gitea/issues/1) as it does not make sense to me to have two conflicting projects that share 99% of the code. As it so often happens with individuals that get overwhelmed by the amount of notifications, this ping got lost and followed a few month of silence and further development in my fork.
Thanks to a ping from @tobiasbp there was traction again and:

🎉  i am happy to share that i will deprecate my fork finally in favor of the official gitea terraform provider 🎉 

All improvements and features have been added to the official provider as well. There are still some minor issues to work out until everything is published to the terraform registry and i will continue to work on the official provider.

Thanks to everyone that used this little provider in the past and submitted PRs als well as reported bugs. All that helps a lot in evolving a software project and i really hope that you will continue to do so for the official provider as well.
The main development will happen on gitea.com (https://gitea.com/gitea/terraform-provider-gitea) but of cause there will always be a github sync as terraform enforces this 😓 

Once the full features are in the official provider i will reference a state migration guide here. the code will probably work the same as the code is pretty much the same :D 

So long and thanks for all the fish 🐬

**Comment by tobiasbp** [comment]
Great news. Thanks for all your work on the provider!
